### PR TITLE
Increased inactivity timeout and never display lights when the displa…

### LINKED ISSUE #207: The Maxmix doesn't turn itself off when not connected to the program.
**Guidelines**
- [x] I have reviewed the [guidelines for contributing](https://github.com/t3knomanzer/maxmix-software/blob/master/.github/CONTRIBUTING.md) to this repository.

**Describe the bug**
The Maxmix Rainbow 🌈 effect that shows when you don't have maxmix connected to the program doesn't stop after turning the pc off, if you have USB set to be powered when PC is off. Essentially the Maxmix isn't put to sleep after x seconds, how it was in the earlier versions. 

**To Reproduce**
1. Go to your computer
2. Click on SHUTDOWN
3. Scroll DOWN with your eyes to the maxmix
4. See rainbow 🌈 

**Expected behavior**
When PC is off, I want my maxmix to not show any kind of light. 

**System information:**
 - OS version: Windows 10
 - Firmware version: 1.3.1

**Additional context**
Love you.

### PR DESCRIPTION
**Pull request by ItsLemmy**
## Issues
 - Fixes #207

## Description
Increased inactivity timeout from 4s to 10s, this should help users with PC that tends to freeze when they open intensive programs.
Reverted to the old behavior : The LEDs are always off when the Oled display is off. This will stop the device from doing its "Circular Funk" effect when the PC is off but the USB port is still powered.
When disconnected, turn on the display and leds a **single** time, so that users can spot when the desktop app crashed.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist
- [x] Requested changes are in a branch
- [x] Compiled and tested requested changes on target hardware (PC, device)
- [ ] Updated the documentation, if necessary
- [ ] Updated the LICENSES file, if necessary
- [x] Reviewed the [guidelines for contributing](../CONTRIBUTING.md) to this repository

### CODE REVIEW

File: Embedded/MaxMix/Config.h [hunk 1]
```diff
@@ -153,4 +153,4 @@ static const uint8_t DISPLAY_GAME_EDIT_CHAR_MAX_WIDTH = DISPLAY_GAME_EDIT_CHAR_M
 static const uint8_t DISPLAY_GAME_WIDGET_VOLUMEBAR_HEIGHT = 7;
 static const uint8_t DISPLAY_GAME_VOLUMEBAR_WIDTH = DISPLAY_AREA_CENTER_WIDTH - DISPLAY_GAME_EDIT_CHAR_MAX_WIDTH - DISPLAY_MARGIN_X2 - 2 - DISPLAY_MARGIN_X1 * 2;
 
-static const uint32_t resetAfterInactivity = 4000;
+static const uint32_t DEVICE_RESET_AFTER_INACTIVTY = 10000;
```
Comment (t3knomanzer) [inline Embedded/MaxMix/Config.h]: I don't think increasing the timeout is going to fix anything. 
People that have reported that the device gets disconnected under some circumstances, such as having their CPU run at 100%.  I was not able to reproduce that at all so there is something else going on with their PCs. But regardless, if their CPU runs at 100% for 10 seconds they will have the same issue. So increasing the time is not really fixing the problem.
As a side effect, now the disconnect detection is not very useful if it takes 10 seconds for the device to realize that the connection has been lost.

Comment (ItsLemmy) [reply Embedded/MaxMix/Config.h]: Would you be ok with 5s ?

Comment (t3knomanzer) [reply Embedded/MaxMix/Config.h]: Makes sense!

Comment (ItsLemmy) [reply Embedded/MaxMix/Config.h]: fixed

**Review by t3knomanzer — APPROVED** [review]

**Pull request merged**